service/collector/tiller_reachable.go

```diff
@@ -19,16 +19,28 @@ var (
 )
 
 func (c *Collector) collectTillerReachable(ctx context.Context, ch chan<- prometheus.Metric) {
```

Comment:
The interface should be aligned with the `exporterkit` collector set. It provides error handling which would make the whole thing a lot easier. We would not need to handle all kinds of weird logging in here but only return errors cleanly.

Reply:
I would rather avoid this in the scope of this PR, as it also involves removing context from parameters and therefore refactoring of `collectChartConfigStatus`.

Reply:
Yeah yeah. All good. I just noticed and made a note. Feel free to get that straight in a follow up PR.